test/test_onnx.py

```diff
@@ -74,6 +74,18 @@ def to_numpy(tensor):
                 else:
                     raise
 
+    def test_new_empty_tensor(self):
+        class Module(torch.nn.Module):
+            def __init__(self):
+                super(Module, self).__init__()
+                self.conv2 = ops.misc.ConvTranspose2d(16, 33, (3, 5))
+
+            def forward(self, input2):
+                return self.conv2(input2)
+
+        input = torch.empty(0)
```

Comment:
Can you make `input` be instead a 4d tensor, with proper shapes?
Like
```python
input = torch.rand(0, 16, 10, 10)
```
this way we make sure that the tests work as expected. Indeed, transposed convolution only support tensors with a batch size of zero, the other dimensions should be valid